### PR TITLE
Rax nodes

### PR DESCRIPTION
**Pull request by nigelbabu**

### CODE REVIEW

File: roles/jenkins_builder/tasks/main.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 - include: setup_disk_{{ location }}.yml
   when: ansible_system == 'Linux' and location
+  tags: key
```
Comment (mscherer) [inline roles/jenkins_builder/tasks/main.yml]: We do not really use tag, I might remove it on merge.

File: roles/jenkins_builder/tasks/main.yml [hunk 2]
```diff
@@ -37,6 +38,16 @@
     name: jenkins
     groups: wheel
 
+- import_tasks: setup_auth_rax.yml
+  when: location == 'rax'
```
Comment (mscherer) [inline roles/jenkins_builder/tasks/main.yml]: Why only for rax ?

Comment (nigelbabu) [reply roles/jenkins_builder/tasks/main.yml]: I want to use different keys for different groups of systems.

File: roles/jenkins_builder/tasks/main.yml (hunk 1 above)
Comment (nigelbabu) [reply roles/jenkins_builder/tasks/main.yml]: Yeah, this was only there for testing. Slipped through. I've removed it already.

**Comment by mscherer** [comment]
ok, gonna merge and apply as soon as I am in the office (so in 3/4h cause lunch break)

**Comment by mscherer** [comment]
Merged it, but:
remote: fatal: [builder100.cloud.gluster.org]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).\r\n", "unreachable": true}


we need to add the key from ansible somehow to be able to manage them from the bastion.

**Pull request closed without merging**